### PR TITLE
[KIECLOUD-542] Conscious Language on KieCloud repos

### PR DESCRIPTION
**Pull request by desmax74**
Signed-off-by: desmax74 <mdessi@redhat.com>
see: https://issues.redhat.com/browse/KIECLOUD-542

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ It also includes the application templates, however they are deprecated in 7.12.
...
       - [jdbc](templates/contrib/jdbc): Examples about how to add third party jdbc driver on RHPAM images.
-    - [docs](templates/docs):  Application templates docs generated by the [gen_template_docs.py](https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/gen-template-doc/gen_template_docs.py) script. \
+    - [docs](templates/docs):  Application templates docs generated by the [gen_template_docs.py](https://github.com/jboss-container-images/jboss-kie-modules/blob/main/tools/gen-template-doc/gen_template_docs.py) script. \
 
 Inside each image directory you will find the following files:
 
  - **container.yaml**: used by OSBS builds.
  - **content_sets.yaml**: define the yum repositories needed to install dependencies for the image.
- - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies. The module.yaml and image.yaml always refer to master branch.
+ - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies. The module.yaml and image.yaml always refer to main branch.
```
Comment (spolti) [inline README.md:28]:
```suggestion
 - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies.
```

File: businesscentral-monitoring/image.yaml [hunk 2]
```diff
@@ -74,27 +74,27 @@ modules:
     - name: cct_module
       git:
         url: https://github.com/jboss-openshift/cct_module.git
-        ref: master
+        ref: main
```
Comment (spolti) [inline businesscentral-monitoring/image.yaml:77]: see my comment on the jira.

File: templates/docs/rhpam712-kieserver-mysql.adoc [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ////
     AUTOGENERATED FILE - this file was generated via
-    https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/gen-template-doc/gen_template_docs.py.
+    https://github.com/jboss-container-images/jboss-kie-modules/blob/main/tools/gen-template-doc/gen_template_docs.py.
```
Comment (spolti) [inline templates/docs/rhpam712-kieserver-mysql.adoc:3]: there's no need to update the adocs files, instead you need to make sure that the script and template files used to generate it is up to date.

**Review by spolti — CHANGES_REQUESTED** [review]

File: templates/docs/rhpam712-kieserver-mysql.adoc (hunk 3 above)
Comment (desmax74) [reply templates/docs/rhpam712-kieserver-mysql.adoc:3]: ok

File: businesscentral-monitoring/image.yaml (hunk 2 above)
Comment (desmax74) [reply businesscentral-monitoring/image.yaml]: ok

**Review by spolti — CHANGES_REQUESTED** [review]
did you forget to update the branches for cct_modules and jboss-eap-modules?

File: businesscentral-monitoring/branch-overrides.yaml [hunk 4]
```diff
@@ -26,8 +26,8 @@ modules:
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: master
+        ref: main
```
Comment (spolti) [inline businesscentral-monitoring/branch-overrides.yaml:29]: isn't changes missing here?

File: businesscentral/branch-overrides.yaml [hunk 5]
```diff
@@ -26,8 +26,8 @@ modules:
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: master
+        ref: main
```
Comment (spolti) [inline businesscentral/branch-overrides.yaml:29]: isn't changes missing here?

File: controller/branch-overrides.yaml [hunk 6]
```diff
@@ -26,8 +26,8 @@ modules:
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: master
+        ref: main
```
Comment (spolti) [inline controller/branch-overrides.yaml:29]: isn't changes missing here?

File: dashbuilder/branch-overrides.yaml [hunk 7]
```diff
@@ -26,8 +26,8 @@ modules:
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: master
+        ref: main
```
Comment (spolti) [inline dashbuilder/branch-overrides.yaml:29]: isn't changes missing here?

File: kieserver/branch-overrides.yaml [hunk 8]
```diff
@@ -26,8 +26,8 @@ modules:
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: master
+        ref: main
```
Comment (spolti) [inline kieserver/branch-overrides.yaml:29]: isn't changes missing here?

File: process-migration/branch-overrides.yaml [hunk 9]
```diff
@@ -14,8 +14,8 @@ modules:
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: master
+        ref: main
```
Comment (spolti) [inline process-migration/branch-overrides.yaml:17]: isn't changes missing here?

File: smartrouter/branch-overrides.yaml [hunk 10]
```diff
@@ -18,8 +18,8 @@ modules:
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: master
+        ref: main
```
Comment (spolti) [inline smartrouter/branch-overrides.yaml:21]: isn't changes missing here?

**Review by spolti — CHANGES_REQUESTED** [review]

File: businesscentral/branch-overrides.yaml (hunk 5 above)
Comment (desmax74) [reply businesscentral/branch-overrides.yaml:29]: done

File: businesscentral-monitoring/branch-overrides.yaml (hunk 4 above)
Comment (desmax74) [reply businesscentral-monitoring/branch-overrides.yaml:29]: done

File: controller/branch-overrides.yaml (hunk 6 above)
Comment (desmax74) [reply controller/branch-overrides.yaml:29]: done

File: dashbuilder/branch-overrides.yaml (hunk 7 above)
Comment (desmax74) [reply dashbuilder/branch-overrides.yaml:29]: done

File: kieserver/branch-overrides.yaml (hunk 8 above)
Comment (desmax74) [reply kieserver/branch-overrides.yaml:29]: done

File: process-migration/branch-overrides.yaml (hunk 9 above)
Comment (desmax74) [reply process-migration/branch-overrides.yaml:17]: done

File: smartrouter/branch-overrides.yaml (hunk 10 above)
Comment (desmax74) [reply smartrouter/branch-overrides.yaml:21]: done

**Review by spolti — APPROVED** [review]

**Pull request merged**